packages/plugins/typescript/operations/src/ts-operation-variables-to-object.ts

```diff
@@ -1,7 +1,33 @@
 import { TypeScriptOperationVariablesToObject as TSOperationVariablesToObject } from '@graphql-codegen/typescript';
 
+const SCALARS = {
+  ID: 'string',
```

Comment:
I believe variables can take `string | number` for IDs.
Here's a reference table: https://the-guild.dev/graphql/hive/blog/the-complete-graphql-scalar-guide#scalar-value-coercion